backend/src/main/java/saci/domain/service/LevelService.java

```diff
@@ -0,0 +1,46 @@
+package saci.domain.service;
+
+import java.util.List;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import saci.domain.model.Level;
+import saci.domain.service.exceptions.AlreadyExistsException;
+import saci.domain.service.exceptions.NotFoundException;
+import saci.infrastructure.LevelRepository;
+
+@Service
+@RequiredArgsConstructor
+public class LevelService {
+
+    private final LevelRepository LevelRepository;
+
+    public Level createLevel(Level Level) {
+        Optional<Level> optionalLevel = LevelRepository.findByName(Level.getName());
+        if (optionalLevel.isPresent()) {
+            throw new AlreadyExistsException("Level name already exists");
+        }
+        return LevelRepository.save(Level);
+    }
+
+    public List<Level> getLevels() {
+        return LevelRepository.findAll();
+    }
+
+    public void deleteLevelById(long LevelId) {
+        Optional<Level> Level = LevelRepository.findById(LevelId);
```

Comment:
```suggestion
        Optional<Level> level = LevelRepository.findById(LevelId);
```

Variables should be camelCase as well